MODULE.bazel

```diff
@@ -16,3 +16,5 @@ bazel_dep(name = "bazel_skylib", version = "1.7.1", dev_dependency = True)
 bazel_dep(name = "rules_pkg", version = "1.0.1", dev_dependency = True)
 bazel_dep(name = "rules_python", version = "0.35.0", dev_dependency = True)
 bazel_dep(name = "stardoc", version = "0.6.2", dev_dependency = True)
+
+register_toolchains("@rules_license//rules_gathering:example_sbom_tools_toolchain")
```

Comment:
Registering a default toolchain is a great move.

Would be nice to have a small doc explaining to users that the toolchain should be registered automatically in BzlMod, and that it can be overridden if folks don't like the default option.